lib/marginalia.rb

```diff
@@ -67,33 +67,38 @@ def annotate_sql(sql)
       sql
     end
 
-    def execute_with_marginalia(sql, name = nil)
-      execute_without_marginalia(annotate_sql(sql), name)
+    def execute_with_marginalia(sql, *args)
```

Comment:
If we change these signatures to `(sql, *args, **kwargs)` (and continue to pass those `kwargs` along down the line), would that allow us to remove the need for `ruby2_keywords` altogether? or maybe that causes a backwards compatibility issue?

Reply:
Yeah, the keyword arg change between Ruby 2.6 and Ruby 2.7 make it so that `ruby2_keywords` is the only way to support blind argument delegation across both versions.

I know it looks dirty, but in the end it's the cleanest solution.